src/main/psp/SDL_psp_main.c

```diff
@@ -28,7 +30,9 @@ PSP_MAIN_THREAD_ATTR(THREAD_ATTR_VFPU | THREAD_ATTR_USER);
 
 int sdl_psp_exit_callback(int arg1, int arg2, void *common)
 {
-    sceKernelExitGame();
+    SDL_Event exit_event;
+    exit_event.quit = (SDL_QuitEvent) {SDL_QUIT, SDL_GetTicks()};
```

Comment:
Please zero the event and fill out the members individually, like the other places we sent SDL_QUIT.

Also, can you target main with new PRs? In this case the event name changed, so it's fine to have this and a separate PR for main, but in general our workflow is making changes in main and then cherry-picking to the SDL2 branches.

Thanks!

Reply:
Okay, my bad. Should I create an MR for SDL3 as well?